src/vkcapture.c

```diff
@@ -98,7 +98,7 @@ typedef struct {
     bool allow_transparency;
     bool window_match;
     bool window_exclude;
-    const char *window;
+const char *window;
```

Comment:
Please fix

Reply:
Whoospy, that slipped in. Reverted it now :bowing_man: